gm4_soul_fragments/data/gm4_soul_fragments/tags/blocks/air.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This should be removed, as the new base adds a #gm4:air block tag. Do a find and replace: "#gm4_soul_fragments:air" with "#gm4:air"